packages/uhk-web/src/app/store/effects/device.ts

```diff
@@ -286,8 +286,8 @@ export class DeviceEffects {
 
     private sendUserConfigToKeyboard(userConfiguration: UserConfiguration, hardwareConfig: HardwareConfiguration): void {
         this.deviceRendererService.saveUserConfiguration({
-            uniqueId: hardwareConfig && hardwareConfig.uniqueId,
-            configuration: userConfiguration.toJsonObject()
-        });
+                                                             uniqueId: hardwareConfig && hardwareConfig.uniqueId,
```

Comment:
Why did such indentations change? I preferred the old style.

Reply:
I upgraded my editor and it was its default settings.